packages/bpk-component-calendar/src/BpkCalendarGrid.js

```diff
@@ -58,14 +56,12 @@ class BpkCalendarGrid extends Component {
         props.month,
         props.weekStartsOn,
       ),
-      daysOfWeek: orderDaysOfWeek(props.daysOfWeek, props.weekStartsOn),
```

Comment:
Same here, especially for localised here where some countries the first day of the week is not the same as UK

Reply:
I don't believe it is used in the rest of the grid.